R/hms_class.R

```diff
@@ -269,3 +269,105 @@ setMethod("printBlockedSprouts", "hms", function(object) {
     cat("\n")
   }
 })
+
+#' Title
+#'
+#' @param object - hms s4 object
+#' @param dimensions - two selected dimensions
+#'
+#' @export
+setGeneric("plotPopulation", function(object, dimensions) standardGeneric("plotPopulation"))
+
+#' Title
+#'
+#' @param object - hms s4 object
+#' @param dimensions - two selected dimensions
+#'
+#' @export
+setMethod("plotPopulation", "hms", function(object, dimensions) {
```

Comment:
Te wykresy da się rysować tylko w 2d, tak? Nie wiem ile to jest roboty, ale myślałem o tym, żeby dało się tutaj podać jeszcze funkcję, która z punktu o dowolnych wymiarach daje punkt w 2d, tak, żeby dało się też po podaniu tej punkcji plotować populacje jak dziedzina ma więcej wymiarów

Reply:
tak działa ta funkcja - argument dimensions może być równy c(3,5) i wtedy z każdego punktu biorę jego 3. i 5. współrzędną

Reply:
myślałem żeby dać też możliwość plotowania w 3D - jest scatterplot w 3D ale nie chciało mi się

Reply:
a oki takie uogólnienie brzmi sensownie, ale na razie zostawię tak jak jest